### PR TITLE
feat: LT-1267 - Filter cases by case officer and assigned user

### PR DESCRIPTION
**Pull request by jfaracik**
* Moved existing filter bars to one generic solution powered by LITE Forms
* Added two new fields to case page for filtering: case officer and assigned user
* Added highlighting to organisation's names when searching by that field
* Used generic pagination function for organisation's page
* Moved Organisations content to LCS
* Removed system queues param from get_queues service
* Fixed AccessibleAutocomplete clear bug
* Added E2E tests

Unrelated cat gif:
Zombie cat - Imgur

### CODE REVIEW

File: assets/css/overrides/_all.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
 @import "input";
 @import "main";
-@import "table";
+@import "select";
+@import "table";
```
Comment (cstobart) [inline assets/css/overrides/_all.scss]: new line

File: assets/css/overrides/_input.scss [hunk 2]
```diff
@@ -1,3 +1,10 @@
 .govuk-input {
 	min-height: 40px!important;
 }
+
+.lite-filter-bar {
+	.govuk-input, .lite-autocomplete__hint, .lite-autocomplete__input, .lite-autocomplete__option {
+		padding-left: govuk-spacing(2);
+		padding-right: govuk-spacing(2);
+	}
+}
```
Comment (cstobart) [inline assets/css/overrides/_input.scss]: new line

File: assets/css/overrides/_select.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	height: 40px!important;
+	border-radius: 0!important;
+	-webkit-appearance: none;
+	padding-left: govuk-spacing(2);
+	padding-right: 16px + govuk-spacing(4);
+	background-image: url("/assets/images/chevron.svg");
+	background-repeat: no-repeat;
+	background-position: right govuk-spacing(2) center;
+	background-size: 16px;
+}
```
Comment (cstobart) [inline assets/css/overrides/_select.scss]: new line

File: organisations/views.py [hunk 4]
```diff
@@ -27,12 +29,27 @@ def get(self, request, **kwargs):
 
         organisations, _ = get_organisations(request, convert_dict_to_query_params(params))
 
+        filters = FiltersBar(
```
Comment (a-bek) [inline organisations/views.py]: Could you add some typing to FIltersBar class, of what arguments it expects?

**Pull request merged**